dict/indonesian.yml

```diff
@@ -0,0 +1,292 @@
+    'KE': [Kenya, Kenya*, Orang Kenya, Nairobi]
+    'KM': [Komoro, Komoro*, Orang Komoro, Moroni]
+    'MG': [Madagaskar, Madagaskar*, Orang Madagaskar, Antananarivo]
+    'MU': [Mauritius, Mauritius*, Orang Mauritius, Port Louis]
+    'MW': [Malawi, Malawi*, Orang Malawi, Lilongwe]
+    'MZ': [Mozambik, Mozambik*, Orang Mozambik, Maputo]
+    'RE': [Reunion, Réunion, Reunion*, Réunion*, Orang Réunionese, Orang Reunionese, Reunionnais]
+    'RW': [Rwanda, Rwanda*, Orang Rwanda, Kigali]
+    'SC': [Seychelles, Seychelles*, Orang Seychelloes, Seychellois]
+    'SO': [Somalia, Somalia*, Orang Somalia,  Mogadishu]
+    'TZ': [Tanzania, Tanzania*, Orang Tanzania, Dodoma, Dar es Salaam]
+    'UG': [Uganda, Uganda*, Orang Uganda, Kampala]
+    'YT': [Mayotte, Mayotte*, Orang Mayotte, Mamoudzou]
+    'ZM': [Zambia, Zambia*, Orang Zambia, Lusaka]
+    'ZW': [Zimbabwe, Zimbabwe*, Orang Zimbabwe, Harare]
+
+  MIDDLE:
+    'AO': [Angola, Angola*, Orang Angola, Luanda]
+    'CD': [Republik Demokratik Kongo, DR Kongo, DRK, Demockratik Republik Kongo, DR Kongo*, Orang Republik Demokratik Kongo, Kinshasa]
+    'CF': [Republik Afrika Tengah, Afrika Tengah, Republik Afrika Tengah*, Afrika Tengah*, Orang Republik Afrika Tengah, Orang Afrika Tengah, Bangui]
+    'CG': [Kongo, Republik Kongo, Kongo*, Republik Kongo*, Orang Kongo, Orang Republik Kongo, Brazzaville]
+    'CM': [Kamerun, Kamerun*, Orang Kamerun, Orang Yaounde, Yaoundé]
+    'GA': [Gabon, Gabon*, Orang Gabon, Libreville]
+    'GQ': [Guinea Khatulistiwa, Guinea Ekuatorial, Guinea Khatulistiwa*, Guinea Ekuatorial*, Orang Guinea Khatulistiwa, Orang Ekuatoguinea, Malabo]
+    'ST': [Sao Tome dan Principe, São Tomé und Príncipe, Sao Tome dan Principe*, São Tomé und Príncipe*, Orang Sao Tome dan Principe]
+    'TD': [Chad, Chad*, Orang Chad, N'Djamena]
+
+  NORTH:
+    'DZ': [Aljazair, Aljazair*, Orang Aljazair, Aljir]
+    'EG': [Mesir, Maeir*, Orang Mesir, Kairo]
+    'EH': [Sahara Barat, Sahara Barat*, Orang Sahara Barat, El Aaiun]
+    'LY': [Libya, Libia, Libya*, Libia*, Orang Libya, Orang Libia, Tripoli]
+    'MA': [Maroko, Maroko*, Orang Maroko, Rabat]
+    'SD': [Sudan, Sudan*, Orang Sudan, Khartoum]
+    'SS': [Sudan Selatan, S Sudan, Sudan Seletan*, Orang Sudan Selatan, Juba]
+    'TN': [Tunisia, Tunisia*, Orang Tunisia, Tunis]
+
+  SOUTH:
+    'BW': [Botswana, Botswana*, Orang Botswana, Gaborone]
+    'LS': [Lesotho, Lesotho*, Orang Lesotho, Maseru]
+    'NA': [Namibia, Namibia*, Orang Namibia, Windhoek]
+    'SZ': [Swaziland, Eswatini, Swaziland*, Eswatini*, Orang Swaziland, Orang Eswatini, Lobamba, Mbabane]
+    'ZA': [Afrika Selatan, S AfriKa, Afrika Selatan*, S AfriKa*, Orang Afrika Selatan, Cape Town, Johannesburg, Pretoria]
+
+  WEST:
+    'BF': [Burkina Faso, Burkina Faso*, Orang Burkina Faso, Ouagadougou]
+    'BJ': [Benin, Benin*, Orang Benin, Porto Novo]
+    'CI': [Pantai Gading, Pantai Gading*, Côte d'Ivoire, Pantai G, Orang Pantai Gading, Yamoussoukro, Abidjan]
```

Comment:
@hirokokinoshita can you check if we really need non-ASCII character like "ô".

Reply:
I would like to keep it English and original spellings are often used.

Reply:
How often do people use English place names? We can combine different languages easily using `c()`. For example,
```
> print(c(data_dictionary_newsmap_en$ASIA, data_dictionary_newsmap_ja$ASIA), 2, -1)
Dictionary object with 5 primary key entries and 2 nested levels.
- [CENTER]:
  - [KG]:
    - kyrgyzstan, kyrgyz*, bishkek, キルギスタン*, キルギス*, ビシュケク
  - [KZ]:
    - kazakhstan, kazakh*, astana, カザフスタン*, カザフ*, アスタナ
  [ reached max_nkey ... 3 more keys ]
- [EAST]:
  - [CN]:
    - china, chinese, beijing, shanghai, 中華人民共和国*, 中国*, 北京, 上海
  - [HK]:
    - hong kong, hongkongese, 香港*
  [ reached max_nkey ... 6 more keys ]
[ reached max_nkey ... 3 more keys ]
```

Reply:
Well, it depends on paper, media, and report, but I suppose English place names are often used. Should I remove all English place name in the dictionary?